frontend/pages/sign-up.tsx

```diff
@@ -0,0 +1,253 @@
+                <FormattedMessage
+                  defaultMessage="Something went wrong while submitting your form."
+                  id="ylNQY0"
+                />
+              )}
+            </p>
+          </div>
+        )}
+        <div className="md:flex md:gap-4">
+          <div className="w-full">
+            <label htmlFor="first-name">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="First name" id="pONqz8" />
+              </p>
+              <Input
+                type="text"
+                name="firstName"
+                id="first-name"
+                placeholder={intl.formatMessage({
+                  defaultMessage: 'Insert your name',
+                  id: 'Q1Q4rT',
+                })}
+                aria-describedby="first-name-error"
+                register={register}
+              />
+            </label>
+            {errors.firstName && (
+              <p
+                id="first-name-error"
+                className="mt-1 ml-2 font-sans text-xs text-red font-regular"
+              >
+                {errors.firstName.message}
+              </p>
+            )}
+          </div>
+          <div className="w-full">
+            <label htmlFor="last-name">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="Last name" id="txUL0F" />
+              </p>
+              <Input
+                type="text"
+                name="lastName"
+                id="last-name"
+                placeholder={intl.formatMessage({
+                  defaultMessage: 'Insert your last name',
+                  id: 'GYkIYE',
+                })}
+                aria-describedby="last-name-error"
+                register={register}
+              />
+            </label>
+            {errors.lastName && (
+              <p id="last-name-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+                {errors.lastName.message}
+              </p>
+            )}
+          </div>
+        </div>
+        <div className="w-full">
+          <label htmlFor="email">
+            <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+              <FormattedMessage defaultMessage="Email" id="sy+pv5" />
+            </p>
+            <Input
+              type="email"
+              name="email"
+              id="email"
+              placeholder={intl.formatMessage({
+                defaultMessage: 'Insert your email',
+                id: 'ErIkUS',
+              })}
+              aria-describedby="email-error"
+              register={register}
+            />
+          </label>
+          {errors.email && (
+            <p id="email-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+              {errors.email.message}
+            </p>
+          )}
```

Comment:
The error is still not displayed if the email is incorrect.

I believe it's because you haven't entirely replicated the input examples from Storybook:
- You need to pass `shouldUseNativeValidation: true` to the `useForm` hook
- You need to add `noValidate` on the `<form />` tag